illumos-utils/src/opte/port_manager.rs

```diff
@@ -326,6 +310,20 @@ impl PortManager {
             (port, ticket)
         };
 
+        // Initialize firewall rules for the new port.
+        let rules = opte_firewall_rules(firewall_rules, &vni, &mac);
+        debug!(
```

Comment:
minor: I know this was `debug!` before, but should such an initialization be considered for `info` logging, akin to others for registration/initialization, etc?

Reply:
All the logging in this method is at `debug!()` level, so I think it makes sense to keep it as-is. There are a few ways one can extract this information if needed.